src/lib/core/data-table/data-table.ts

```diff
@@ -161,7 +145,7 @@ export class CdkTable<T> implements CollectionViewer {
   }
 
   ngOnDestroy() {
-    this._onDestroy.next();
+    this._onDestroy.complete();
```

Comment:
I think `next()` is still required for `takeUntil`([example](http://plnkr.co/edit/Rjxn1GDA8RbXUtzWYIhc?p=preview))

I suspect @ErinCoughlan was suggesting you use both